.gitignore

```diff
@@ -4,3 +4,4 @@
 composer.lock
 node_modules/
 Modules/
+/.project
```

Comment:
I see, sorry, it was the "automatic" habit of my workplace. Now removed and moved all that stuff on the global `.gitignore`
@mikemand Many thanks for make me noticing of this, had no idea! :grinning: